devel/cloc.xml

```diff
@@ -101,4 +127,7 @@
       <file dest="cloc.exe" href="https://github.com/AlDanial/cloc/releases/download/1.86/cloc-1.86.exe" size="9414945"/>
     </implementation>
   </group>
+  <package-implementation distributions="Gentoo" package="dev-util/cloc"/>
+  <package-implementation distributions="Ports" package="misc/cloc"/>
+  <package-implementation package="cloc"/>
```

Comment:
Could you add `main="/usr/bin/cloc"` attributes to these `<package-implementation>`s?

Reply:
done